app/templates/components/donations/donation-status.hbs

```diff
@@ -0,0 +1,7 @@
+{{#if subscription}}
+  {{donations/show-donation amount=subscription.amount}}
+{{else if processStarted}}
+  {{donations/create-donation subscription=subscription continue=(action createDonation) amount=15}}
```

Comment:
Curious, how can we pass `subscription` here if it isn't present? If `subscription` exists, then the first `if` condition will be met and this component won't be rendered.  This component is only shown if _don't_ have a subscription.